lib/src/arithmetic/ones_complement_adder.dart

```diff
@@ -25,16 +25,18 @@ class OnesComplementAdder extends Adder {
   @protected
   Logic _sign = Logic();
 
-  /// [OnesComplementAdder] constructor with an adder functor [adderGen]
-  /// Either a Logic [subtractIn] or a boolean [subtract] can enable
-  /// subtraction, with [subtractIn] overriding [subtract].  If Logic [carryOut]
-  /// is provided as not null, then the end-around carry is not performed and is
-  /// left to the caller via the output [carryOut].
+  /// [OnesComplementAdder] constructor with an adder functor [adderGen].
+  ///  - Either a Logic [subtractIn] or a boolean [subtract] can enable
+  /// subtraction, with [subtractIn] overriding [subtract].
```

Comment:
should `subtract` and `subtractIn` both be `null` by default so that you can detect if both are configured for an exception?